views/confirmation/income.pug

```diff
@@ -0,0 +1,40 @@
+extends ../base
+
+block variables
+  -var title = __('Confirm 2019 Income')
+
+block content
+
+  h1 #{title}
+
+  if(data.login.ageYesNo === 'Yes')
+    p #{__('If your income is above $19,564:')}
+  else
+    p #{__('If your income is above $12,069:')}
```

Comment:
Let's default to ` $19,564` if the value doesn't exist or is wrong. 

This strict check means that if something isn't right here, it will show the lower value, which is probably wrong one to show.